packages/subgraph/networks.json

```diff
@@ -0,0 +1,8 @@
+{
+  "goerli": {
+    "PoolMetadataRegistry": {
+      "address": "0xEBfADf723E077c80f6058dC9c9202bB613DE07cf",
+      "startBlock": 8583917
+    }
+  }
+}
```

Comment:
is this intended to generate manifests automatically? what's the script that does this?